### ISSUE TITLE
[Feature Request] All workspaces active/alive from the start, not only after they are selected (WITH WORKAROUND)

### ISSUE THREAD

**Issue by goncaloalves** [glzr-io/glazewm#784]
### Describe the problem/motivation

Only workspaces with windows are active from the start.

### Describe the solution you'd like

It would be nice if all workspaces could be active/alive from the start, not only after they are selected as in other Window Managers (ex. i3)

## Current workaround that I am using
Change your startup commands to something like this: startup_commands: ['shell-exec zebar', "move --workspace 2", "move --workspace 6", "move --workspace 1"]

It also works really well with the tip from issue 720:
```html
<button
                      className={`workspace ${workspace.hasFocus && 'focused'} ${workspace.isDisplayed && 'displayed'} ${workspace.children.length && 'has-windows'}`}
                      onClick...>...</button>
```

Here is an image of this at work right after startup (underlined workspace means it has windows):
![image](https://github.com/user-attachments/assets/bbf1650e-6889-4eb3-98c7-0b560dd73c2b)

**Comment by DreamMaoMao** [comment]
To be honest, I don't understand what you mean

**Comment by goncaloalves** [comment]
When GlazeWM starts, the only workspaces loaded are the ones currently displayed (monitor 1 has 1, monitor 2 has 1, etc.). Comparing this to i3 for instance, all workspaces are loaded at the start. This means that you can display them in Zebar right from the start.

I think it should be possible to have all workspaces loaded from the start even if they don't have any window currently.

Hope this clarifies it.

**Comment by DreamMaoMao** [comment]
![image](https://github.com/user-attachments/assets/bebf400e-4dbb-4cba-a445-e13318230507)
Hasn't this feature always existed?

**Comment by goncaloalves** [comment]
Not really. Right now the workspaces are only active after you placed a window in them or manually select them via a keyboard shortcut. 

This feature request would make all workspaces active at the start of glazeWM.

Right now I'm using a workaround that switches to all workspaces at the start of glazeWM. It would be a nice feature if this workaround wasn't necessary.

**Comment by DreamMaoMao** [comment]
I understand that this is to set the initial state of each workspace.

**Comment by goncaloalves** [comment]
I was thinking I could try and tackle this feature request.

I think there are two main approaches to this problem:
1 - Have a config option that allows the user to select if the workspace is active at the start of GlazeWM and start those workspaces at start
2 - Create a new IPC command that lists all the workspaces (active and inactive) and each user customizes his own zebar. I think there are some memory/ performance benefits to this approach at the expense of a more difficult configuration.

I think I know how to implement the second (2) approach. But I would like to get some feedback on this, if this should be implemented at all and what is the best approach. It would be ideal if the maintainers of GlazeWM could answer these questions.

Would also be nice if other people could also request this feature, before I spend time coding a feature that is not necessary :)

**Comment by goncaloalves** [comment]
my understanding is that this command only returns workspaces that have been activated before. On startup if I run this command and only have one monitor, this command will return one workspace. Is this correct?

I want it to return all workspaces that are in the config file.

**Comment by DreamMaoMao** [comment]
The first method, which is not easy to handle, assumes that for multiple monitor users, each workspace should be initially bound to which monitor is uncertain, until it is activated on one monitor, it will trigger the binding, and the second case is very simple to just add a command to return to all workspaces, but the zebar side should also be changed. I haven't looked at the zebar code yet
